yarn.lock

```diff
@@ -291,8 +291,8 @@
   resolved "https://registry.yarnpkg.com/@types/faker/-/faker-4.1.2.tgz#f8ab50c9f9af68c160dd71b63f83e24b712d0df5"
 
 "@types/history@*":
-  version "4.6.2"
-  resolved "https://registry.yarnpkg.com/@types/history/-/history-4.6.2.tgz#12cfaba693ba20f114ed5765467ff25fdf67ddb0"
+  version "4.7.0"
```

Comment:
`react-router` depends on this.
And, `History.Location` was changed that have generics.
see [here](https://github.com/DefinitelyTyped/DefinitelyTyped/compare/91f8a6b7d1ef61cc543d0901dcc004b846283345%E2%80%A6f11a3342486e62b9f62a0fe54263a744bb5a34a0#diff-c219a7e1afec470d8818be362072a58aR27)